MANIFEST.in

```diff
@@ -48,6 +37,5 @@ recursive-include docs Makefile
 recursive-include misc *.py
 recursive-include misc *.rst
 recursive-include tests *.py
-
-# added by check_manifest.py
 recursive-include invenio_i18n *.po
```

Comment:
Ah, yes we should include ``*.mo`` files (i.e. excluded from git, but included in python package) - it should also be added to cookiecutter.